cmd/entrypoint-wrapper/main.go

```diff
@@ -41,6 +41,13 @@ var (
 	encoder runtime.Encoder
 )
 
+const (
+	skipKubeconfigFlag   = "skip-kubeconfig"
+	rwKubeconfigFlag     = "readwrite-kubeconfig"
```

Comment:
do we need this as it is default right now?

Reply:
Yes, `rw` kubeconfig is the default but +1 for the catch, it is not needed for observers. I made a commit to set it to false, thanks!